apps/btshell/src/cmd_leaudio.c

```diff
@@ -17,13 +17,13 @@
  * under the License.
  */
 
-#include "host/ble_audio_broadcast_source.h"
 #include "cmd_leaudio.h"
 #include "btshell.h"
 #include "console/console.h"
 #include "errno.h"
 
-#if (MYNEWT_VAL(BLE_ISO_BROADCAST_SOURCE))
+#if (MYNEWT_VAL(BLE_AUDIO))
+#include "audio/ble_audio_bcst_source.h"
```

Comment:
I'm not very happy with this bcst abbreviation...

Reply:
So maybe ble_audio_broadcast_src? and later we'll use snk counterpart

Reply:
Left full `broadcast` in names